### PR TITLE
Refactored imports like described in our contributing doc.

### PR DESCRIPTION
**Pull request by TimonPost**
Refactored imports like we discussed in our contribution doc (not available for public yet).

I'll merge this directly since it has no changes in functionality only in how the code looks. And we already discussed that part.

### CODE REVIEW

**Comment by TimonPost** [comment]
bors +r

File: src/protocol_version.rs [hunk 1]
```diff
@@ -1,10 +1,10 @@
-use std::sync::Mutex;
-
 use crc::crc32;
 use lazy_static;
 
 pub use net::constants::PROTOCOL_VERSION;
 
+use std::sync::Mutex;
```
Comment (LucioFranco) [inline src/protocol_version.rs]: I believe this use should go above the pub use.

Comment (TimonPost) [reply src/protocol_version.rs]: According to how we, at least me and Fletcher, agreed on some format about how we should order our imports. And this is the way it should be as described in that document. I've sent it in discord.

**Comment by TimonPost** [comment]
I need this to merge for my other PR's I am going to make due conflicts.

bors +r

**Comment by TimonPost** [comment]
bors +r

**Comment by TimonPost** [comment]
I need this to merge for my other PR's I am going to make due conflicts.

bors r+

**Comment by TimonPost** [comment]
`error: toolchain 'beta' is not installed` You know why this is @LucioFranco ?

**Comment by TimonPost** [comment]
Oke lets try it again bors r+

**Comment by TimonPost** [comment]
Oke lets try it again 

bors r+

**Comment by fhaynes** [comment]
Travis is having issues with the beta toolchain on macOS. I'm temporarily allowing those to fail to get these PRs done.

**Comment by fhaynes** [comment]
Which I can't do here because of the fork the repo model. Sigh.

**Comment by TimonPost** [comment]
Going to close it because it has and will have more conflicts if #95 gets merged.

**Pull request closed without merging**